packages/contracts/src/tasks/routerFactory_create.ts

```diff
@@ -0,0 +1,27 @@
+import { task } from "hardhat/config";
+
+export default task("create-router", "create a router")
+  .addParam("signer", "The router's signer address")
+  .addParam("recipient", "recipient address")
+  .setAction(async ({ signer, recipient }, { deployments, getNamedAccounts, ethers }) => {
+    const namedAccounts = await getNamedAccounts();
+
+    console.log("signer: ", signer);
+    console.log("recipient: ", recipient);
+
+    console.log("namedAccounts: ", namedAccounts);
+
+    const routerFactoryDeployment = await deployments.get("RouterFactory");
+    const routerFactoryAddress = routerFactoryDeployment.address;
+
+    console.log("routerFactoryAddress: ", routerFactoryAddress);
+
+    const routerFactory = await ethers.getContractAt("RouterFactory", routerFactoryAddress);
```

Comment:
can we check that the router factory didn't already deploy a contract at that address before we try to recreate it?